spec/trimmer_decorator_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../class/trimmer_decorator'
+
+describe TrimmerDecorator do
+  before :each do
+    @dummy_trimmer_decorator = TrimmerDecorator.new('Fickry Bil Iman')
+  end
+
+  describe '-> Initialize class (.new)' do
+    it 'should be create an instance of TrimmerDecorator class' do
+      expect(@dummy_trimmer_decorator).to be_an_instance_of(TrimmerDecorator)
+    end
+  end
+end
```

Comment:
- [ ] The correct_name method should actually  be tested here and should trim the person's name from 0..9 :+1:

Reply:
fixes the method and the unit tests